### PR TITLE
Feature/add annualized performance

### PR DESCRIPTION
**Pull request by dtslvr**

### CODE REVIEW

File: apps/api/src/app/portfolio/portfolio.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      null,
+      null,
+      null
+    );
+  });
+
+  /**
+   * Source: https://www.investopedia.com/terms/a/annualized-total-return.asp#annualized-return-formula-and-calculation
+   */
+  fit('Get annualized performance', async () => {
```
Comment (vzickner) [inline apps/api/src/app/portfolio/portfolio.service.spec.ts:23]: The f shouldn't be here

File: apps/api/src/app/portfolio/portfolio.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    ).toEqual(0);
+
+    expect(
+      portfolioService.getAnnualizedPerformancePercent({
+        daysInMarket: 0,
+        netPerformancePercent: 0
+      })
+    ).toEqual(0);
+
+    expect(
```
Comment (vzickner) [inline apps/api/src/app/portfolio/portfolio.service.spec.ts:38]: Maybe another test with 365 days?

File: apps/api/src/app/portfolio/portfolio.service.ts [hunk 3]
```diff
@@ -80,6 +81,21 @@ export class PortfolioService {
...
+  public getAnnualizedPerformancePercent({
+    daysInMarket,
+    netPerformancePercent
+  }: {
+    daysInMarket: number;
+    netPerformancePercent: number;
+  }) {
+    if (isNumber(daysInMarket) && daysInMarket > 0) {
+      const exponent = new Big(365).div(daysInMarket).toNumber();
+      return Math.pow(1 + netPerformancePercent, exponent) - 1;
```
Comment (vzickner) [inline apps/api/src/app/portfolio/portfolio.service.ts:93]: Why not with big?

Comment (dtslvr) [reply apps/api/src/app/portfolio/portfolio.service.ts:93]: `pow` only works with `int` in `Big.js` → https://mikemcl.github.io/big.js/#pow

File: apps/api/src/app/portfolio/portfolio.service.spec.ts (hunk 2 above)
Comment (dtslvr) [reply apps/api/src/app/portfolio/portfolio.service.spec.ts:38]: Good idea! 👍🏻

File: apps/api/src/app/portfolio/portfolio.service.spec.ts (hunk 1 above)
Comment (dtslvr) [reply apps/api/src/app/portfolio/portfolio.service.spec.ts:23]: Eagle eye! Thank you @vzickner 🙂

File: apps/api/src/app/portfolio/portfolio.service.ts (hunk 3 above)
Comment (vzickner) [reply apps/api/src/app/portfolio/portfolio.service.ts:93]: Sad

**Pull request merged**